### PR TITLE
feature(homepage) mention browser support of webpack

### LINKED ISSUE #2000: Document browser compatibility
https://github.com/webpack/webpack/pull/6993

### PR DESCRIPTION
**Pull request by EugeneHlushko**
* mention browser support (on the homepage or other place To be discussed)

Fixes #2000

### CODE REVIEW

File: src/content/index.md [hunk 1]
```diff
@@ -67,5 +67,8 @@ Then run `webpack` on the command-line to create `bundle.js`.
 
 __[Get Started](/guides/getting-started)__ quickly in our __Guides__ section, or dig into the __[Concepts](/concepts)__ section for more high-level information on the core notions behind webpack.
 
+## Browser Compatibility
+webpack supports all browsers that are [ES5-compliant](http://kangax.github.io/compat-table/es5/) (IE8 and below are not supported). webpack needs `Promise` for `import()` and `require.ensure()`. If you want to support older browsers, you will need to [load a polyfill](https://webpack.js.org/guides/shimming/) before using these expressions.
```
[the remark this answers is not in the document]
Comment (EugeneHlushko) [reply src/content/index.md]: sorry i missed it, will update

File: src/content/index.md [hunk 2]
```diff
@@ -67,5 +67,8 @@ Then run `webpack` on the command-line to create `bundle.js`.
 
 __[Get Started](/guides/getting-started)__ quickly in our __Guides__ section, or dig into the __[Concepts](/concepts)__ section for more high-level information on the core notions behind webpack.
 
+## Browser Compatibility
+webpack supports all browsers that are [ES5-compliant](https://kangax.github.io/compat-table/es5/) (IE8 and below are not supported). webpack needs `Promise` for `import()` and `require.ensure()`. If you want to support older browsers, you will need to [load a polyfill](/guides/shimming/) before using these expressions.
```
Comment (dhruvdutt) [inline src/content/index.md]: Should external links (ES5 -compliant) open in new tab/target blank?

Comment (dhruvdutt) [inline src/content/index.md]: Should external links (ES5 -compliant) open in new tab/target blank?

Comment (jeremenichelli) [inline src/content/index.md]: Can you add a line break after the title. Then, it would be good to merge to me.

**Comment by EugeneHlushko** [comment]
Thanks! Fixed it @jeremenichelli

**Comment by jeremenichelli** [comment]
Asking for another review just because is the index page 😊

@montogeek what do you think?

**Comment by montogeek** [comment]
I am not sure about adding it in the homepage, Maybe a subpage would be a better fit? Configuration maybe?

**Comment by jeremenichelli** [comment]
Maybe as an ending note in **Concepts**, not that it belongs as a concept but is the landing page when you click **Documentation** link.

**Comment by EugeneHlushko** [comment]
Hmm i agree with @jeremenichelli that ending note in concepts makes sense, if you are happy with that @montogeek i will apply the change

**Comment by montogeek** [comment]
@EugeneHlushko Sure!

**Comment by EugeneHlushko** [comment]
@montogeek @jeremenichelli guys this been here for a while, any chance to merge it (just updated it again)

**Comment by montogeek** [comment]
Need @jeremenichelli approval

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**